### ISSUE TITLE
[bug]: Default value for `max-commit-fee-rate-anchors` is too low

### ISSUE THREAD

**Issue by tnull** [lightningnetwork/lnd#8240]
### Background

We had a user run into issues as LND is capping the anchor commitment fee at `MaxAnchorsCommitFeeRate`, which defaults 		to `lnwallet.DefaultAnchorsCommitMaxFeeRateSatPerVByte` which is set to 10 sat/vb since https://github.com/lightningnetwork/lnd/pull/4855. As the mempool minimum nowadays often spikes above that value, LND will try to use a fee rate that won't allow commitment transactions to propagate, which LDK on the other hand won't allow.

While the `sample-lnd.conf` makes note of this as: `Must be large enough to ensure transaction propagation`, the current default seems unreasonably low.

### Your environment

* version of `lnd`: v0.17.1-beta (should be the same for master/v0.17.2-beta)

**Comment by Roasbeef** [comment]
Given the current fee climate, I think it makes sense to increase the default value. While we're at it, we should add a new RPC that allows callers to update the default value and/or queue up and trigger an `update_fee` message.

**Comment by rbndg** [comment]
This would be very helpful for us, since we use LDK on mobile and we use LND for LSP

**Comment by Crypt-iQ** [comment]
> We had a user run into issues as LND is capping the anchor commitment fee at MaxAnchorsCommitFeeRate, which defaults to lnwallet.DefaultAnchorsCommitMaxFeeRateSatPerVByte which is set to 10 sat/vb since https://github.com/lightningnetwork/lnd/pull/4855. As the mempool minimum nowadays often spikes above that value, LND will try to use a fee rate that won't allow commitment transactions to propagate, which LDK on the other hand won't allow.

Is this during the opening of the channel? During channel operation, LND's commitment fee _should_ go higher than the minimum relay fee here: https://github.com/lightningnetwork/lnd/blob/ac9ca024a4bfb2890e2cd450666bd94d221fe172/lnwallet/channel.go#L8464

**Comment by feelancer21** [comment]
I agree that the default of 10 sat/vb is far too low. This currently leads to SCB restores peers possibly not sending their commit TX if the channel was opened by the peer and he did not care about the parameter.
The max anchor commit fee should therefore be higher than the purging rate for a standard mempool assuming a high-fee environment. I would therefore suggest increasing the parameter to ~200 sat/vb.

**Comment by litbot-9000** [comment]
Consolidating related issue here. The following has been closed as a duplicate:

- #8299 — Reports `max-commit-fee-rate-anchors` being completely ignored during force close. Concrete example: setting was configured to 25 sat/vB, but the force close tx was published at 589 sat/vB (~24x the configured limit), resulting in ~$100 in fees. This was on lnd 0.17.3 with a WOS-initiated force close where the reporter was the channel opener. References PR #8096 as the expected fix.
